src/db/dao/CharacterNotificationDao.ts

```diff
@@ -20,10 +21,16 @@ export default class CharacterNotificationDao {
       .groupBy("characterNotification_type")
       .run()
       .then((rows) =>
-        _.map(rows, function (row) {
+        _.map(rows, async function (row) {
+          const sols = [
+            ...row.characterNotification_text.matchAll(/solarSystemID: (\d+)/),
```

Comment:
This feels pretty jargon-y, especially along with the indices down below. Maybe use named groups instead, and just check if a match object exists? If you only ever care about the first match, why are you using `matchAll` instead of `match`?

Reply:
fixed.